packages/node_modules/@ciscospark/bin-sauce-connect/bin/sauce-connect

```diff
@@ -1,18 +1,19 @@
 #!/usr/bin/env node
 
-require('babel-register');
-var path = require('path');
-require('dotenv').config({
-  path: path.join(__dirname, '../../../.env')
+const path = require(`path`);
+require(`dotenv`).config({
+  path: path.join(__dirname, `../../../../../.env`)
```

Comment:
because this command isn't necessarily run from the root. i think. you're right, but i also know i needed this at some point.  :)